sdk/ml/azure-ai-ml/tests/feature_store/e2etests/test_feature_store.py

```diff
@@ -0,0 +1,50 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from typing import Callable
+
+import pytest
+from devtools_testutils import is_live
+
+from azure.ai.ml import MLClient
+from azure.ai.ml.entities._feature_store.feature_store import FeatureStore
+from azure.core.polling import LROPoller
+
+
+@pytest.mark.e2etest
+@pytest.mark.data_experiences_test
+@pytest.mark.usefixtures(
+    "mock_workspace_arm_template_deployment_name", "mock_workspace_dependent_resource_name_generator"
+)
+class TestFeatureStore:
```

Comment:
is it possible to add a test case for testing creating featurestore with offline store and materiazlition UAI?

Reply:
yes, I will work on details e2e tests in other PR. this is just to get the basic stuff.